.gitignore

```diff
@@ -1,21 +1,26 @@
-# dependencies
+#These files are mostly auto-included by npx crate-react-app. Below are minor explanations of what they each are. Further answers can be found by googling "react [thing-name-here]"
+
+
+# dependencies for react -- dont need to be inlcuded, gets included with npm install 
 front-end/slothstagram/node_modules
 /.pnp
 .pnp.js
 
-# testing
+# testing files auto included in npx create. Jest testing framework (so far not using)
 /coverage
 
-# production
+# directory where production builds are built. It contains correctly bundled React in production mode and optimized builds.
 /build
 
-# misc
+# .DS_Store gets generated by macOS users when viewing files in Finder. Not needed.
+# These are environment files that should not be included remotely for security.
 .DS_Store
 .env.local
 .env.development.local
 .env.test.local
 .env.production.local
```

Comment:
Let's ditch the extra `.env.whatever` files and only have `.env`. Make the DS_Store comment only above the DS_Store rule, and same for env comment and rule